setup.py

```diff
@@ -155,9 +155,14 @@
 ]
 extras_require = {
     ':sys_platform != "win32"': ['terminado>=0.3.3'],
-    'doc': ['Sphinx>=1.1'],
+    'doc': [
+        'Sphinx>=1.1',
```

Comment:
But we need the added dependencies for the shown instructions to work!

namely:

```
pip install -e .[test] --user
pip install -e .[doc] --user
```

I can make a separate PR if you prefer, but those changes are directly related to the documentation changes I made.